mayhem/Mayhemfile

```diff
@@ -0,0 +1,9 @@
+project: uncurl
```

Comment:
Please omit the image name (this gets added as part of the action)

Reply:
just removed it

Reply:
when I omitted the image name, it causes the mayhem run to fail. I added it back and the mayhem run succeeds. Should I still leave it out or should i put it back in?

Reply:
It looks like your image tag is referencing an old image. There are two images on this repo:
https://github.com/EthanM27/uncurl/pkgs/container/uncurl

`master` was updated a few minutes ago, which is likely your most recent build. `latest` has not been updated in 2 months. When I pull `master`, I get the following failure:

```
 === Uncaught Python exception: ===
TypeError: ConsumeString(): incompatible function arguments. The following argument types are supported:
    1. (self: atheris.native.FuzzedDataProvider, arg0: int) -> object

Invoked with: <atheris.native.FuzzedDataProvider object at 0x7f87d3766270>
Traceback (most recent call last):
  File "/src/mayhem/fuzz_uncurl.py", line 23, in fuzzUncurlParse
    data = fdp.ConsumeString()

==8== ERROR: libFuzzer: fuzz target exited
SUMMARY: libFuzzer: fuzz target exited
MS: 0 ; base unit: 0000000000000000000000000000000000000000


artifact_prefix='./'; Test unit written to ./crash-da39a3ee5e6b4b0d3255bfef95601890afd80709
Base64: 
```

Reply:
I think with the ConsumeString function, it might require an integer to indicate the length: https://github.com/google/atheris

Maybe I can get the length of what data was fuzzed and pass that to consume string?

Reply:
You can use `remaining_bytes()` for this:
https://github.com/google/atheris/blob/770866ac436ece9aa2e9562f2a3f06086e2d8373/src/native/fuzzed_data_provider.h#L122

For example:
```
mystring = fdp.ConsumeString(fdp.remaining_bytes())
```

Reply:
Ok i made the change. Let me know if there is anything else I need to change